xtokens/src/lib.rs

```diff
@@ -83,6 +83,9 @@ pub mod module {
 		#[pallet::constant]
 		type SelfLocation: Get<MultiLocation>;
 
+		/// Minimum xcm execution fee paid on destination chain.
+		type MinXcmFee: GetOptionValueByKey<MultiLocation, u128>;
```

Comment:
```suggestion
		type MinXcmFee: GetValueByKey<MultiLocation, Option<u128>>;
```

I don't think we need a version for option